### ISSUE TITLE
genhtml: Negative repeat count does nothing

### ISSUE THREAD

**Issue by xaizek** [linux-test-project/lcov#17]
I get a warning from perl interpreter for each processed file:
```
Processing file src/utils/globs.c
genhtml: Negative repeat count does nothing at /usr/bin/genhtml line 3854, <SOURCE_HANDLE> line 162.
```
It points to the last line of this function:
```perl
sub fmt_centered($$)
{
	my ($width, $text) = @_;
	my $w0 = length($text);
	my $w1 = int(($width - $w0) / 2);
	my $w2 = $width - $w0 - $w1;

	return (" "x$w1).$text.(" "x$w2); # <-- here
}
```

It happens with `genhtml_line_field_width = 8` in configuration file because `"Line data"` string has length of 9 in this invocation:
```perl
	my $line_heading = fmt_centered($line_field_width, "Line data");
```

It doesn't affect functionality in any way, but clutters the output.

**Comment by oberpar** [comment]
Thanks for reporting. Your analysis is correct. I haven't seen this before because the warning was introduced with Perl 5.21. The issue should be fixed with commit 6ec3f2398d22e605c1a8019541fb32d26d18044b.

**Comment by xaizek** [comment]
That fixes it, thank you.
